src/main/java/cubicchunks/world/CubeWorldEntitySpawner.java

```diff
@@ -151,88 +130,64 @@ private int spawnCreatureTypeInAllChunks(EnumCreatureType mobType, ICubicWorldSe
             int blockY = blockpos.getY();
             int blockZ = blockpos.getZ();
 
-            int currentPackSize = 0;
```

Comment:
I'm not even trying to figure out what you did here, but it seems like you removed a few potentially important loops. Including the part where MC attemots to spawn mobs in groups.